css/style.css

```diff
@@ -16,4 +19,71 @@ body {
  * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
  */
 
+ main {
+    text-align: center;
+}
+
+footer {
+    text-align: center;
+    display: inline;
+}
+
+ul.bar li {
+    display: inline;
+    list-style-type: none;
+    text-align: right;
+}
+
+
+.cta {
+    font-size: large;
+    color: white;
+    height: 40px;
+    width: 150px;
```

Comment:
It's best to avoid giving fixed widths and heights to buttons, so they can grow with their content. Instead, add more padding to give breathing space inside the button and make it grow while still staying proportionate to the text inside it.